### ISSUE TITLE
Search bar pop-up appears as a full window on Sway

### ISSUE THREAD

**Issue by go3stin** [linuxmint/nemo#2362]
```
 * Nemo version (nemo --version) : 4.4.3
 * Is issue with desktop or windowed nemo? windowed
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Archlinux  (on 5.6.6)
 * Graphics hardware *and* driver used : Intel HD Graphics 630
 * 32 or 64 bit : 64
 ```

**Issue**

Whenever I use local search (typing any letter to start searching), the pop-up window is full windowed. It seems that this behavior has already been encountered by other sway users but it kind of came back and forth without any serious explanation.

**Steps to reproduce**

Simply opening nemo on sway and hitting a few keys.

**Expected behaviour**

Instead of a full window, the search bar should be a pop-up.

**Other information**

Screenshot:

![image](https://user-images.githubusercontent.com/64328012/80288151-468f4680-8736-11ea-9cfd-81037605b720.png)

**Comment by xdnicoco** [comment]
Confirmed. It looks similar to https://bugzilla.xfce.org/attachment.cgi?id=9827 so maybe it has a similar solution.

**Comment by go3stin** [comment]
> Confirmed. It looks similar to https://bugzilla.xfce.org/attachment.cgi?id=9827 so maybe it has a similar solution.

Perfect. I experienced a similar issue with Thunar, but in a much more buggy way: the pop-up (in the form of a full window) would open and close very quickly as I pressed keys.

**Comment by mtwebster** [comment]
Is this only happening with the icon view?

**Comment by go3stin** [comment]
I wouldn't say because I only use the icon view.

**Comment by mtwebster** [comment]
Could you check?

**Comment by go3stin** [comment]
To be honest, I had to switch back to X11 and had been waiting a long time to restructure my partitions... Therefore I won't be able to check that (plus I don't want to mislead you) :/

**Comment by xdnicoco** [comment]
> Is this only happening with the icon view?

It happens in all views, not just icon view.

**Comment by WhyNotHugo** [comment]
Hi. I opened #2380 since I didn't find this first. That ticket has a bit more description too, but it the same issue.

- It happens on icon view.
- It happens on compact view.
- On list view, there's no window. I can't see what I've typed anywhere though, I'm not sure if that's expected behaviour.

I'm happy to test further and help debug, I've been using wayland/sway (and nemo!) for a long time now and don't intend on switching.
